style/styles.css

```diff
@@ -0,0 +1,130 @@
+@import url("./fonts.css");
+@import url("./fonts.css");
+
+* {
+  margin: 0;
+}
+.container {
+  max-width: 1100px;
```

Comment:
You limited the container with a width less than the normal content width from the design (1440px). Look what happens with block `content` which children drop out from it.